### PR TITLE
Add PokemonGo bot version to docker image

### PR DESCRIPTION
**Pull request by walkerlee**
## Short Description:

add commit version when create docker image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,17 +35,18 @@ RUN apk -U --no-cache add python py-pip \
...
     && cp /tmp/pgoencrypt/src/libencrypt.so /usr/src/app/encrypt.so \
     && apk del .pgoencrypt-dependencies \
     && rm -rf /var/cache/apk/* /tmp/pgoencrypt /tmp/pgoencrypt.tar.gz
 
-ADD https://github.com/$BUILD_REPO/archive/$BUILD_BRANCH.tar.gz /tmp
-RUN apk -U --no-cache add --virtual .tar-deps tar \
-    && cat /tmp/$BUILD_BRANCH.tar.gz | tar -zxf - --strip-components=1 -C /usr/src/app \
-    && apk del .tar-deps \
-    && rm /tmp/$BUILD_BRANCH.tar.gz
+ADD https://api.github.com/repos/$BUILD_REPO/git/refs/heads/$BUILD_BRANCH /tmp/pgobot-version
```
Comment (th3w4y) [inline Dockerfile]: this is not working if i specify  a particular commit as the BUILD_BRANCH

Comment (walkerlee) [inline Dockerfile]: is there any situation we need to use a commit?

**Comment by Gobberwart** [comment]
I was identified as a reviewer but I don't use docker so pass.

File: Dockerfile (hunk 1 above)
Comment (th3w4y) [inline Dockerfile]: it order to test new changes... that is why you can also override the BUILD_REPO

In the past there was a COPY . /usr/src/app

the user was able to do it's changes and docker build again

I replaced that with ADD https://github.com/$BUILD_REPO/archive/$BUILD_BRANCH.tar.gz /tmp for various reasons:
- security COPY .  means including auth and unfits... and stuff
- being able to reproduce the build if you know create date  and LABEL build_branch and build_repo and not some local modification that the user might have done...

**Comment by th3w4y** [comment]
@Gobberwart  and now that you commented on this PR the change to be identified next time increased 🎉

File: Dockerfile (hunk 1 above)
Comment (walkerlee) [inline Dockerfile]: ok, so I'll delete this PR

**Comment by Gobberwart** [comment]
LOL and now you commented....

OK I think we're done :D

**Comment by th3w4y** [comment]
version can always be extrapolated from:

docker inspect --format='{{.Created}} {{.ContainerConfig.Labels}}' container_tag_or_id

then use the info and get the commit via git-rev-parse

```
git rev-parse '@{2016-08-25 18:30:30}'
9605728255f996f156fd00b1467d7f14afb1d3a4
```

**Comment by walkerlee** [comment]
@th3w4y I think this way isn't very precisely

**Comment by th3w4y** [comment]
@walkerlee I agree but is the only idea i have for now

**Comment by th3w4y** [comment]
getting the commit seems to work:
https://api.github.com/repos/PokemonGoF/PokemonGo-Bot/commits/9605728255f996f156fd00b1467d7f14afb1d3a4

https://api.github.com/repos/PokemonGoF/PokemonGo-Bot/commits/dev returns same as 
or https://api.github.com/repos/PokemonGoF/PokemonGo-Bot/commits/a388c105ee93734824da32eab87c599e04feab74

since the HEAD of dev at this moment is a388c105ee93734824da32eab87c599e04feab74

**Comment by walkerlee** [comment]
@th3w4y 
LGTM, thank you!

**Comment by th3w4y** [comment]
LABEL BUILD_COMMIT $(curl -H "Accept: application/vnd.github.VERSION.sha" https://api.github.com/repos/PokemonGoF/PokemonGo-Bot/commits/dev)

**Pull request closed without merging**